README.md

```diff
@@ -24,18 +24,19 @@ RustPython requires Rust latest stable version (e.g 1.43.0 at May 24th 2020).
 To check Rust version: `rustc --version` If you wish to update,
 `rustup update stable`.
 
-To test RustPython, do the following:
+To build RustPython locally, do the following:
 
     $ git clone https://github.com/RustPython/RustPython
     $ cd RustPython
-    # if you're on windows:
+      # if you're on windows:
```

Comment:
is this indent intended?

Reply:
Yeah, I think it makes it more clear that it's not just a root prompt/differentiates it from the commands